network/iroha/simplenetwork/docker-compose.yml

```diff
@@ -1,8 +1,8 @@
 version: '3'
 
 services:
-  iroha_node0:
-    image: hyperledger/iroha-docker@sha256:b82a24eee7cc4215bf70d9f79acab8b817e4cf36cc9f87c1c63202dca552cf0f
+  iroha-node0:
+    image: hyperledger/iroha:x86_64-develop-latest
```

Comment:
We (Iroha team) plan to change Iroha's docker [naming](https://soramitsu.atlassian.net/browse/IR-1259) in the nearest future, so now this PR uses image `hyperledger/iroha:x86_64-develop-latest` without any sha256.

When we rename it I'll do commit in this branch with the new name of container on dockerhub.